src/i18n/en-US/intake.ts

```diff
@@ -391,10 +423,13 @@ const intake = {
     businessNeed: 'What is your business need?',
     solving: 'How are you thinking of solving it?',
     process: 'Where are you in the process?',
-    usesAiTech: 'Does your request involve AI technologies?',
     eaSupport: 'Do you need Enterprise Architecture (EA) support?',
+    usesAiTech: 'Does your request involve AI technologies?',
     hasUiChanges:
       'Does your project involve any user interface component, or changes to an interface component?',
+    usingSoftware:
+      'Do you plan to use any software products to fulfill your business needs?',
```

Comment:
this text exists on line 333, can it be reused?

Reply:
I thought about it, but we have usually had one i18n for the form and one for the admin review. The language is usually slightly different between the two but isn't here.

Im fine with using the form label here but it would go against the paradigm that the rest of the questions/answers follow.